README.md

```diff
@@ -268,14 +320,15 @@ ansible-playbook -i hosts ./your-playbook.yml
 
 ### Installing X-Pack Features
 
-X-Pack features, such as Security, are supported. This feature is currently experimental.
+X-Pack features, such as Security, are supported.
 
-The parameter `es_xpack_features` by default enables all features i.e. it defaults to ["alerting","monitoring","graph","security","ml"]
+The parameter `es_xpack_features` allows to list xpack features to install (example: `["alerting","monitoring","graph","security","ml"]`).
+When the list is empty, it install all features available with the current licence.
```

Comment:
This variable doesn't actually install anything. The only time `es_xpack_features` is actually referred to is this line:

https://github.com/elastic/ansible-elasticsearch/blob/0e61dba1ce88ecd37e5178f8c54ae8beaa021dd9/tasks/elasticsearch-parameters.yml#L18-L21

Now that it is empty by default I'm not sure if this adds any value at all. Instead I think we can just drop this whole variable and make sure that we explain that you need to set the username and password if you are using security.

Reply:
It's strange that you saw a change in this PR because this is a change from https://github.com/elastic/ansible-elasticsearch/pull/560/files which is already merged